.. _acknowledgements:

Acknowledgements
================

Source Databases
~~~~~~~~~~~~~~~~~~

`Pathogenwatch <https://pathogen.watch/>`_

`Enterobase <https://enterobase.warwick.ac.uk/>`_

Typing tools
~~~~~~~~~~~~~~~~~~

*Klebsiella pneumoniae*
----------------------

Genotype nomenclature
^^^^^^^^^^^^^^^^^^^^^

- 7-locus MLST scheme: `Multilocus Sequence Typing of Klebsiella pneumoniae Nosocomial Isolates, Diancourt 2005 <https://doi.org/10.1128/jcm.43.8.4178-4182.2005>`_

- Core genome MLST scheme: `Genomic Definition of Hypervirulent and Multidrug-Resistant Klebsiella pneumoniae Clonal Groups, Bialek-Davenet 2014 <https://doi.org/10.3201/eid2011.140206>`_

- Sublineages: `A Dual Barcoding Approach to Bacterial Strain Nomenclature: Genomic Taxonomy of Klebsiella pneumoniae Strains, Hennart 2022 <https://doi.org/10.1093/molbev/msac135>`_

- MLST databases: `BIGSdb Institut Pasteur <https://bigsdb.pasteur.fr/klebsiella/>`_

AMR typing
^^^^^^^^^^

- Kleborate v3: `https://github.com/klebgenomics/Kleborate <https://github.com/klebgenomics/Kleborate>`_

- Kleborate paper: `A genomic surveillance framework and genotyping tool for Klebsiella pneumoniae and its related species complex, Lam 2021 <https://doi.org/10.1038/s41467-021-24448-3>`_

*Klebsiella* Pathogenwatch: `Rapid Genomic Characterization and Global Surveillance of Klebsiella Using Pathogenwatch, Argímon 2021 <https://doi.org/10.1093/cid/ciab784>`_



*Escherichia coli* 
------------------

Genotype nomenclature
^^^^^^^^^^^^^^^^^^^^^

- Achtman 7-locus MLST scheme: Developed by `Enterobase <https://enterobase.readthedocs.io/en/latest/mlst/mlst-legacy-info-ecoli.html/>`_

- Core genome MLST scheme: `The EnteroBase user's guide, with case studies on Salmonella transmissions, Yersinia pestis phylogeny, and Escherichia core genomic diversity, Zhou et al 2020 <http://www.genome.org/cgi/doi/10.1101/gr.251678.119>`_

- Hierarchical clustering: `HierCC: a multi-level clustering scheme for population assignments based on core genome MLST, Zhou et al 2020 <https://doi.org/10.1093/bioinformatics/btab234>`_

- MLST databases: `EnteroBase <https://enterobase.warwick.ac.uk/>`_

Pathotype definition
^^^^^^^^^^^^^^^^^^^^^

- The diarrheagenic *E. coli* (DEC) pathotypes are defined using `Enterobase pathotyping logic <https://enterobase.readthedocs.io/en/latest/pipelines/backend-pipeline-phylotypes.html?highlight=pathovar>`_

- *Shigella* species identification : `Hierarchical Cluster (HC) <https://enterobase.readthedocs.io/en/latest/pipelines/backend-pipeline-phylotypes.html?highlight=pathovar>`_

AMR typing
^^^^^^^^^^^^^^^^^^^^^

- `AMRFinderPlus <https://github.com/ncbi/amr>`_



*Neisseria gonorrhoeae* 
-----------------------

Genotype nomenclature
^^^^^^^^^^^^^^^^^^^^^

-7-locus MLST scheme: `Species status of Neisseria gonorrhoeae: evolutionary and epidemiological inferences from multilocus sequence typing, Bennett et al 2007 <https://doi.org/10.1186/1741-7007-5-35>`_

-2-locus N. gonorrhoeae multi-antigen sequence typing (NG-MAST) scheme `Rapid Sequence-Based Identification of Gonococcal Transmission Clusters in a Large Metropolitan Area, Martin et al 2004 <https://doi.org/10.1086/383047>`_

- MLST and NG-MAST databases: `PubMLST <https://pubmlst.org/neisseria/>`_

AMR typing
^^^^^^^^^^^^^^^^^^^^^

- Pathogenwatch in-house module as described in the paper `A community-driven resource for genomic epidemiology and antimicrobial resistance prediction of Neisseria gonorrhoeae at Pathogenwatch, Sánchez-Busó et al. 2021 <https://doi.org/10.1186/s13073-021-00858-2>`_



*Salmonella* (non-typhoidal)
-----------------------------

Genotype nomenclature
^^^^^^^^^^^^^^^^^^^^^

- 7-locus MLST scheme: `Enterobase <https://enterobase.readthedocs.io/en/latest/mlst/mlst-legacy-info-senterica.html>`_

- Core genome MLST scheme: `The EnteroBase user's guide, with case studies on Salmonella transmissions, Yersinia pestis phylogeny, and Escherichia core genomic diversity, Zhou et al 2020 <http://www.genome.org/cgi/doi/10.1101/gr.251678.119>`_

- Hierarchical clustering: `HierCC: a multi-level clustering scheme for population assignments based on core genome MLST, Zhou et al 2020 <https://doi.org/10.1093/bioinformatics/btab234>`_

- Serotypes: `The Salmonella In Silico Typing Resource (SISTR): An Open Web-Accessible Tool for Rapidly Typing and Subtyping Draft Salmonella Genome Assemblies, Yoshida et al 2016 <https://doi.org/10.1371/journal.pone.0147101>`_

AMR typing
^^^^^^^^^^^

- `AMRFinderPlus <https://github.com/ncbi/amr>`_
