.. _acknowledgements:

Acknowledgements
================

Source Databases
~~~~~~~~~~~~~~~~~~

`Pathogenwatch <https://pathogen.watch/>`_

`Enterobase <https://enterobase.warwick.ac.uk/>`_

Typing tools
~~~~~~~~~~~~~~~~~~

*Klebsiella pneumoniae* genotype nomenclature

- 7-locus MLST scheme: `"Multilocus Sequence Typing of Klebsiella pneumoniae Nosocomial Isolates", Diancourt 2005 <https://doi.org/10.1128/jcm.43.8.4178-4182.2005>`_

- Core genome MLST scheme: `"Genomic Definition of Hypervirulent and Multidrug-Resistant Klebsiella pneumoniae Clonal Groups", Bialek-Davenet 2014 <https://doi.org/10.3201/eid2011.140206>`_

- Sublineages: `"A Dual Barcoding Approach to Bacterial Strain Nomenclature: Genomic Taxonomy of Klebsiella pneumoniae Strains", Hennart 2022 <https://doi.org/10.1093/molbev/msac135>`_

- MLST databases: `BIGSdb Institut Pasteur <https://bigsdb.pasteur.fr/klebsiella/>`_

*Klebsiella pneumoniae* AMR typing

- Kleborate v3: `https://github.com/klebgenomics/Kleborate <https://github.com/klebgenomics/Kleborate>`_

- Kleborate paper: `"A genomic surveillance framework and genotyping tool for Klebsiella pneumoniae and its related species complex", Lam 2021 <https://doi.org/10.1038/s41467-021-24448-3>`_

*Klebsiella* Pathogenwatch: `"Rapid Genomic Characterization and Global Surveillance of Klebsiella Using Pathogenwatch", Arg√≠mon 2021 <https://doi.org/10.1093/cid/ciab784>`_
