
User guide
==========
.. raw:: html


   <div class="justify-text">
      AMRnet aims to make high-quality, robust and reliable genome-derived
      AMR surveillance data accessible to a wide audience. Visualizations are
      geared towards showing national annual AMR prevalence estimates and trends,
      that can be broken down and explored in terms of underlying genotypes and
      resistance mechanisms. We do not generate sequence data, but we hope
      that by making publicly deposited data more accessible and useful,
      we can encourage and motivate more sequencing and data sharing.
   </div>

`Subscribe <https://amrnet.beehiiv.com/subscribe>`_ to the ``AMRnet`` mailing list!

Installation
------------

You can install ``AMRnet`` via `pip <https://pip.pypa.io/>`_ from
`PyPI <https://pypi.org/>`_:

.. code:: shell

   pip install AMRnet

.. See :ref:`label-installation` for more information.

.. Features
.. --------

.. See the complete list of ``AMRnet`` features :ref:`label-features`.

.. Contributing
.. ------------

.. Contributions are very welcome. To learn more, see the :ref:`label-contributing`.

License
-------

Distributed under the terms of the :doc:`GPLv3 License <license>` ``AMRnet``
is free and open source software.

Support
-------

Found a bug üêõ, or have a feature request ‚ú®, raise an issue on the
GitHub `issue
tracker <https://github.com/amrnet/amrnet/issues>`_.
Alternatively you can get support on the
`discussions <https://github.com/amrnet/amrnet/discussions>`_
page.

Disclaimer
----------
.. raw:: html

      <div class="justify-text">
         AMRnet is an open source dashboard that continues to benefit from
         the collaboration of many contributors. Although efforts have been made to ensure the
         that relevant web development standards are implemented, it remains the
         user's responsibility to confirm and accept the output. Refer to the
         :doc:`License <license>` for clarification of the conditions of use.
      </div>

Documentation
=============
.. raw:: html
   
      <div class="justify-text">
         The <a href="https://www.amrnet.org/" target="_blank" class="amr-link"> AMRnet dashboard </a> aims to make high-quality, robust and reliable genome-derived AMR surveillance data accessible to a wide audience. Visualizations are geared towards showing national annual AMR prevalence estimates and trends, that can be broken down and explored in terms of underlying genotypes and resistance mechanisms. We do not generate sequence data, but we hope that by making publicly deposited data more accessible and useful, we can encourage and motivate more sequencing and data sharing.
         <div class="spacer"></div>We started with Salmonella Typhi, built on our The <a href="https://www.typhi.net" target="_blank" class="amr-link">TyphiNET</a> dashboard which uses data curated by the <a href="http://typhoidgenomics.org" target="_blank" class="amr-link">Global Typhoid Genomics Consortium </a> (to improve data quality and identify which datasets are suitable for inclusion) and analysed in <a href='http://pathogen.watch' target='_blank'>Pathogenwatch</a> (to call AMR determinants and lineages from sequence data). More organisms will be added throughout 2024-25, using data sourced from analysis platforms such as <a href='http://pathogen.watch' target='_blank'>Pathogenwatch</a>, <a href='https://enterobase.warwick.ac.uk/' target='_blank'>Enterobase</a>, and potentially others.
         <div class="spacer"></div>A major barrier to using public data for surveillance is the need for careful data curation, to identify which datasets are relevant for inclusion in pooled estimates of AMR and genotype prevalence. This kind of curation can benefit a wide range of users and we plan to work with other organism communities to curate data, and to contribute to wider efforts around metadata standards. Please get in touch if you would like to work with us (<a href="mailto:amrnetdashboard@gmail.com">amrnetdashboard@gmail.com</a>)
         <div class="spacer"></div>Find out more about the project team (based at London School of Hygiene and Tropical Medicine), and our policy advisory group, <a href="https://www.lshtm.ac.uk/amrnet" target="_blank">here</a>.
         <div class="spacer"></div>The dashboard code is open access and available in <a href="https://github.com/amrnet/amrnet" target="_blank">GitHub</a>. Issues and feature requests can be posted <a href="https://github.com/amrnet/amrnet/issues" target="_blank">here</a>. API access is described on the <a href="https://amrnet.readthedocs.io/en/latest/data.html" target=''>Data access</a> page.
      </div>

Citation for AMRnet
-------------------
.. raw:: html
   
      <div class="justify-text">
         If you use the AMRnet website or code, please cite AMRnet (Louise Cerdeira, Vandana Sharma, Kat Holt), <span class="bold-text">GitHub</span> <a href="https://github.com/amrnet/amrnet" target="_blank">https://github.com/amrnet/amrnet</a>, <span class="bold-text">DOI</span>: 10.5281/zenodo.10810219
      </div>