AMRnet
======

.. container:: justify-text

   The `AMRnet <https://www.amrnet.org/>`_ aims to make high-quality, robust and reliable genome-derived AMR surveillance data accessible to a wide audience. Visualizations are geared towards showing national annual AMR prevalence estimates and trends, that can be broken down and explored in terms of underlying genotypes and resistance mechanisms. We do not generate sequence data, but we hope that by making publicly deposited data more accessible and useful, we can encourage and motivate more sequencing and data sharing.

   We started with Salmonella Typhi, built on our `TyphiNET <https://www.typhi.net>`_ dashboard which uses data curated by the `Global Typhoid Genomics Consortium <http://typhoidgenomics.org>`_ (to improve data quality and identify which datasets are suitable for inclusion) and analysed in `Pathogenwatch <http://pathogen.watch>`_ (to call AMR determinants and lineages from sequence data). Dashboards are now available for Klebsiella pneumoniae, Neisseria gonorrhoeae, non-typhoidal Salmonella, E. coli and Shigella using data sourced from the `Pathogenwatch <http://pathogen.watch>`_ and `Enterobase <https://enterobase.warwick.ac.uk/>`_ platforms. In the future, we hope to add additional organisms, sourced from these and other platforms.

   A major barrier to using public data for surveillance is the need for careful data curation, to identify which datasets are relevant for inclusion in pooled estimates of AMR and genotype prevalence. This kind of curation can benefit a wide range of users and we plan to work with other organism communities to curate data, and to contribute to wider efforts around metadata standards. Please get in touch if you would like to work with us (`amrnetdashboard@gmail.com <amrnetdashboard@gmail.com>`_)

   Find out more about the project team (based at London School of Hygiene and Tropical Medicine), and our policy advisory group, `here <https://www.lshtm.ac.uk/amrnet>`__.

   The dashboard code is open access and available in `GitHub <https://github.com/amrnet/amrnet>`_. Issues and feature requests can be posted `here <https://github.com/amrnet/amrnet/issues>`__. API access is described on the :doc:`/data` page.

Citation for AMRnet
-------------------

.. container:: justify-text

   If you use the AMRnet website or code, please cite AMRnet (Louise Cerdeira, Vandana Sharma, Kathryn Holt), **GitHub**: https://github.com/amrnet/amrnet, **DOI**: 10.5281/zenodo.10810219



Dashboard code
==============
.. container:: justify-text

   If you want to install the AMRnet code to develop your own dashboard instances, you can install ``AMRnet`` via `pip <https://pip.pypa.io/>`_ from
   `PyPI <https://pypi.org/>`_:

      .. code:: shell

         pip install AMRnet


License
-------
.. container:: justify-text

    Distributed under the terms of the :doc:`GPLv3 License <license>`.

   ``AMRnet`` is free and open source software.

Support
-------
.. container:: justify-text

   Found a bug üêõ, or have a feature request ‚ú®, raise an issue on the
   GitHub `issue
   tracker <https://github.com/amrnet/amrnet/issues>`_.
   Alternatively you can get support on the
   `discussions <https://github.com/amrnet/amrnet/discussions>`_
   page.

Disclaimer
----------
.. container:: justify-text

   ``AMRnet`` is an open source dashboard that continues to benefit from
   the collaboration of many contributors. Although efforts have been made to ensure the
   that relevant web development standards are implemented, it remains the
   user's responsibility to confirm and accept the output. Refer to the
   :doc:`License <license>` for clarification of the conditions of use.
